hedera-node/hedera-app/src/main/java/com/hedera/node/app/grpc/impl/netty/NettyGrpcServerManager.java

```diff
@@ -269,7 +269,7 @@ private NettyServerBuilder builderFor(final int port, NettyConfig config) {
                     .bossEventLoopGroup(new EpollEventLoopGroup())
                     .workerEventLoopGroup(new EpollEventLoopGroup());
             logger.info("Using Epoll for gRPC server");
-        } catch (final Exception ignored) {
+        } catch (final UnsatisfiedLinkError | NoClassDefFoundError ignored) {
```

Comment:
How do you know that it can only be one of these?

Reply:
We tested it.
Also, These are the errors specified by the JLS to be thrown when a native library cannot be found or when a Java library class cannot be found, which are the possible errors if EPoll isn't available.